website/get-involved/index.html

```diff
@@ -34,6 +35,25 @@ <h2>{{ self.page_desc() }}</h2>
 </nav>
 
 <main role="main">
+    <section id="opensource">
+      <h2>{{ _('Why Contribute?') }}</h2>
+      <div class="row">
+            <div class="col full">
+                <p>{{ _('Thunderbird is created by an online community of volunteers.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('What is Open Source Software?') }}</h3>
+                    <p>{{ _('Thunderbird is open source software, meaning that anyone can inspect, modify, enhance it, and share it. But the fun is not just for programmers!
+                        Thunderbird is created by a community of designers, translators, people who do documentation and support, as well as developers. We invite anyone to come
+                        and participate in the creation of Thunderbird regardless of your skillset, we’re sure there is something you can add to our community 
```

Comment:
This is a bit of a run-on, I think. "We invite anyone to come and participate in the creation of Thunderbird. Regardless of your skillset, we're sure there is something you can add to our community and project." sounds better to me.

Reply:
I agree. Will change.